Assets/MixedRealityToolkit.Staging/README.md

```diff
@@ -0,0 +1,11 @@
+# What is MixedRealityToolkit.Staging
+
+The MixedRealityToolkit.Staging folder in the Microsoft Mixed Reality Toolkit contains a collection of components that are in active development and
```

Comment:
I think that this README misses the major point of this folder, which is that this is a staging/temporary one that is intended to be removed in the weeks following, and is only here to avoid a risky set of changes to packaging.

Especially interesting is to ensure that folks aren't taking a dependency on the structure of this (i.e. it will find a different home shortly)

Reply:
sounds good. i will elaborate a bit more.

Reply:
Adding:


ng changes that may have posed risk to the product.

> [!Important]
> This folder is expected to be removed from the **_next_** version of the Microsoft Mixed Reality Toolkit. Please take no dependencies on the current structure.

Reply:
I think it would be helpful to reword this and put the second part first (i.e. this is a staging folder). 

The fact that it contains useful stuff is a generally true statement (i.e. if it didn't contain useful stuff, it shouldn't exist in this repo).

Reply:
So actually maybe then just removing the first part of it would be all it takes here.

Reply:
fixed.